modules/govuk/manifests/node/s_api_lb.pp

```diff
@@ -60,6 +61,14 @@
     internal_only => true,
   }
 
+  if !empty($mapit_servers) {
+    loadbalancer::balance { 'mapit':
+      servers       => $mapit_servers,
+      internal_only => true,
+      https_only    => false, # FIXME: Remove for #51136581
```

Comment:
Can we remove this line? If there's nothing that will be making requests to mapit on port 80 it's not required :)